gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Tue Aug 19 17:12:51 CDT 2014
```

Comment:
Not if you update forge properly, it's only overwritten by the gradle wrapper task.